### PR TITLE
Suggestions

### PR DESCRIPTION
**Pull request by terceiro**
hi @utkarsh2102. I have read through the current source and it looks good so far. This PR has some suggestions for improvement.

### CODE REVIEW

File: lib/rubocop/cop/packaging/gemspec_git.rb [hunk 1]
```diff
@@ -38,9 +38,10 @@ module Packaging
       #   good_foo_method(args)
       #
       class GemspecGit < Cop
-        MSG = "Don't use git."\
-              "It's a problem while maintaining it downstream."\
-              "Rather use some pure Ruby alternative, like `Dir` or `Dir.glob`"
+        MSG = "Avoid using git to produce lists of files." \
+          "Downstreams often need to build your package in an environment " \
+          "that does not have git (on purpose)." \
+          "Use some pure Ruby alternative, like `Dir` or `Dir.glob`"
```
Comment (utkarsh2102) [inline lib/rubocop/cop/packaging/gemspec_git.rb:44]: Hi,
Thanks for this. However, while testing this locally, it renders like this:
image
Although I can do it myself, but while you're at it, can you add spaces at the end of the sentences and a fullstop at the end?

And also use single-quotes instead? :see_no_evil: 
This throws a bunch of "offenses":
```
lib/rubocop/cop/packaging/gemspec_git.rb:41:15: C: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
        MSG = "Avoid using git to produce lists of files." \
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/rubocop/cop/packaging/gemspec_git.rb:42:11: C: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
          "Downstreams often need to build your package in an environment " \
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/rubocop/cop/packaging/gemspec_git.rb:43:11: C: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
          "that does not have git (on purpose)." \
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/rubocop/cop/packaging/gemspec_git.rb:44:11: C: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
          "Use some pure Ruby alternative, like `Dir` or `Dir.glob`"
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

**Comment by terceiro** [comment]
the ci seems to be broken on a missing dependency

**Comment by utkarsh2102** [comment]
> the ci seems to be broken on a missing dependency

Yeah, I need to fix a bunch of things because of adding `tasks/` yesterday.
I'll do this bit and the documentation part. Even `generate_cops_documentation` is broken.

Anyway, thanks for this! :taco:

**Comment by utkarsh2102** [comment]
> > the ci seems to be broken on a missing dependency
> 
> Yeah, I need to fix a bunch of things because of adding `tasks/` yesterday.

This is done, CI is green again! :smile:

**Pull request merged**